qemu/tests/cpu_device_hotplug_during_boot.py

```diff
@@ -21,14 +23,16 @@ def run(test, params, env):
     """
     vcpu_devices = params.objects("vcpu_devices")
     unplug_during_boot = params.get_boolean("unplug_during_boot")
+    vcpus_count = params.get_numeric("vcpus_count", 1)
```

Comment:
What does vcpus_count mean? Is it for ppc ?

Reply:
It is equivalent to vcpus_count in "query-hotpluggable-cpus".
```
{"execute": "query-hotpluggable-cpus"}
{"return": [{"props": {"core-id": 0, "node-id": 0}, "vcpus-count": 1, "qom-path": "/machine/unattached/device[1]", "type": "host-spapr-cpu-core"}]}
```

Reply:
Thanks for the clarification.